config/settings.yml

```diff
@@ -988,6 +988,12 @@ lighthouse:
       rsa_key: ~
       aud_claim_url: https://deptva-eval.okta.com/oauth2/ausln2mo4jCAYRrlR2p7/v1/token
     use_mocks: false
+  benefits_intake:
+    host: https://sandbox-api.va.gov
+    path: /services/vba_documents
+    version: v1
+    use_mocks: false
+    api_key: ~
```

Comment:
Will you be adding this information to the manifest repo (and parameter store)? https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-Secretvalues

Reply:
@rmtolmach Yes absolutely, wanted to get the library up for review before tackling the deployment (ie. needed to know what the configuration would be first)